### PR TITLE
Ensure mypostscript has permission 700

### PR DESCRIPTION
**Pull request by mattaezell**
Only root needs to execute this file

### CODE REVIEW

**Comment by whowutwut** [comment]
@immarvin can you review the changes here?

File: xCAT-server/share/xcat/install/scripts/post.debian [hunk 1]
```diff
@@ -60,6 +60,7 @@ do
          rm -rf $i
          chmod +x /xcatpost/*
          /xcatpost/getpostscript.awk |egrep '<data>' |sed -e 's/<[^>]*>//g'|egrep -v '^ *$'|sed -e 's/^ *//' | sed -e 's/&lt;/</g' -e 's/&gt;/>/g' -e 's/&amp;/\&/g' -e 's/&quot;/"/g' -e "s/&apos;/'/g" > /xcatpost/mypostscript
+         chmod 700 /xcatpost/mypostscript 2>/dev/null
```
Comment (immarvin) [inline xCAT-server/share/xcat/install/scripts/post.debian]: hi @mattaezell , I think this PR should only replace the strings like ``chmod +x mypostscript`` or ``chmod 777 mypostscript`` to ``chmod 700 mypostscript``, do not need to chmod the requested /xcatpost/mypostscript

Comment (mattaezell) [reply xCAT-server/share/xcat/install/scripts/post.debian]: My goal was to reduce the amount of time that the _mypostscript_ file has 755 permissions.  It should be very brief, so I guess I can remove these lines if you think they are superfluous.

**Comment by immarvin** [comment]
hi @mattaezell , get your point. My though is that before mypostscript is ready run, the content of requested mypostscript is either the error response or raw data,  I suggest to remove the lines, sorry for the inconvenience, thanks

File: xCAT-server/share/xcat/install/scripts/post.debian (hunk 1 above)
Comment (neo954) [reply xCAT-server/share/xcat/install/scripts/post.debian]: @mattaezell, for this purpose, you may add `umask 0077` in somewhere in this file.
Maybe, change line 62 to `( umask 0077 ; /xcatpost/getpostscript.awk |egrep '<data>' |sed -e 's/<[^>]*>//g'|egrep -v '^ *$'|sed -e 's/^ *//' | sed -e 's/&lt;/</g' -e 's/&gt;/>/g' -e 's/&amp;/\&/g' -e 's/&quot;/"/g' -e "s/&apos;/'/g" > /xcatpost/mypostscript )`
Thus, only one line is affected with `umak 0077`. Then line 63 can be omitted.

**Comment by immarvin** [comment]
I am ok with the change, agree to merge

**Pull request merged**